app/assets/stylesheets/modules/_results-section.scss

```diff
@@ -42,3 +42,30 @@
   margin: 0;
 }
 
+.paging {
+  clear: both;
+  padding: 20px;
+  text-align: center;
+  width: 100%;
+  margin: 0 auto;
+
+  a {
+    display: block;
```

Comment:
Properties should be ordered background-color, border-radius, color, display, height, margin, padding, text-align, width